src/refactor_nrepl/ns/libspec_whitelist.clj

```diff
@@ -0,0 +1,49 @@
+(ns refactor-nrepl.ns.libspec-whitelist
```

Comment:
Let's avoid the problematic "whitelist" terminology. I guess the namespace can be simply named "libspec".

Reply:
The nice thing about the name is that it maps 1:1 to the existing piece of config.

FWIW I also favor avoiding these terminologies where I can, but often keeping things maintainable/non-broken takes precedence.

Reply:
I know how the config value is named, but I don't see any reason to name a namespace after a configuration option. Especially after in the long run it's highly likely that the current name will be deprecated in favor of some alias with a different name.